### DISCUSSION TITLE
Pre qualify criteria (PQC) in on_search response from the lenders

### DISCUSSION THREAD

**Discussion by ankitagarwaladobe** [ONDC-Official/ONDC-FIS-Specifications#30] [General]
Can we enable lenders to share their PQC in the on_search responses?

The lenders can respond with multiple generic offers and corresponding PQC. This can help the BAPs to target the users accordingly based on the signals they have.

**Comment by Antriksh-Parmar** [comment]
Let's begin with a simpler implementation in 0-1 and gradually add additional use cases and complexities.

**Reply by ankitagarwaladobe** [reply]
Sure, it's your call but we feel that this would be simpler implementation and not a complex use case.
The PQC in on_search response could be optional for the lenders to respond with.

**Reply by ankitagarwaladobe** [reply]
This is inline with this https://github.com/ONDC-Official/ONDC-FIS-Specifications/discussions/9#discussioncomment-6779938
what we discussed in person as well.
Happy to discuss this further.
